admin/helpers/routes.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose')
+
+const School = require('../models/school')
+const Pupil = require('../models/pupil')
+const Answer = require('../models/answer')
+
+const fetchPupilsData = async (req, next) => {
```

Comment:
@jon-shipley check my latest commit and let's discuss it from there. Even though the fetchPupilsData returns an object with a lean and a non-lean mongoose object it's quite convenient as in most cases both are required to be used.